app/controllers/api/v1/road_trip_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::RoadTripController < ApplicationController
+  def create
+    user = User.find_by(api_key: params[:api_key])
```

Comment:
This is the "Gordian Knot" solution to a problem I thought about for some time - there must be a better way to authenticate a user with an api key than by just looking them up by that key and checking if they exist. However, that likely requires more security than this app currently has (such as making the api_key column a token rather than a string).